File: apps/problems/two-sum/boilerplate-full/function.java

import java.io.*;
import java.util.*;

public class Main {
    
    ##USER_CODE_HERE##

    public static void main(String[] args) {
        // The judge pipes the testcase in on stdin.
        List<String> lines = readLinesFromStdin();
        int num1 = Integer.parseInt(lines.get(0).trim());
  int num2 = Integer.parseInt(lines.get(1).trim());
        int result = sum(num1, num2);
        System.out.println(result);
    }

    public static List<String> readLinesFromStdin() {
        List<String> lines = new ArrayList<>();
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String line;
            while ((line = br.readLine()) != null) {
                lines.add(line);
            }
        } catch (IOException e) {
            e.printStackTrace();
        }
        return lines;
    }
}

File: apps/problems/two-sum/boilerplate-full/function.js
##USER_CODE_HERE##

// The judge pipes the testcase in on stdin; fd 0 is that pipe.
const input = require('fs').readFileSync(0, 'utf8').trim().split('\n').join(' ').split(/\s+/);
const num1 = parseInt(input.shift());
  const num2 = parseInt(input.shift());
const result = sum(num1, num2);
console.log(result);
    